src/api_server/Cargo.toml

```diff
@@ -43,6 +44,8 @@ tokio.workspace = true
 tonic = { version = "0.8", optional = true }
 uuid = { version = "1.2.2", features = ["serde", "v4"] }
 openssl = { version = "0.10.46", optional = true }
+reqwest = { version = "0.11.13", features = ["json"], optional = true }
```

Comment:
Please move the two new dependencies following lexicographical order, thanks!

Reply:
I have updated the patch.